pkg/security/podsecuritypolicy/seccomp/strategy_test.go

```diff
@@ -21,25 +21,31 @@ import (
 	"strings"
 	"testing"
 
+	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	api "k8s.io/kubernetes/pkg/apis/core"
 )
 
 var (
-	withoutSeccomp    = map[string]string{"foo": "bar"}
+	foo               = "foo"
+	bar               = "bar"
+	withoutSeccomp    = map[string]string{foo: bar}
```

Comment:
it's confusing to use foo and bar here... just do `"some-annotation":"some-value"`?

Reply:
Reverted that change.